src/sst/core/impl/interactive/simpleDebug.cc

```diff
@@ -71,7 +71,7 @@ SimpleDebugger::cmd_pwd(std::vector<std::string>& UNUSED(tokens))
 void
 SimpleDebugger::cmd_ls(std::vector<std::string>& UNUSED(tokens))
 {
-    std::vector<std::pair<std::string, SST::Core::Serialization::ObjectMap*>> vars = obj_->getVariables();
+    auto& vars = obj_->getVariables();
```

Comment:
Shouldn't use auto here, but we can change it back in a future PR so we can get the majority of the changes merged.